notifier.cpp

```diff
@@ -13,26 +13,26 @@ struct myPubMsgInfo {
     char        ID[30];
 };
 
-static volatile bool done = false;
+//static volatile bool done = false;
 // natsOptions* opts   = NULL;
 // const char* cluster    = "cyberway";
 // const char* clientID   = "notifier";
 // const char* subj   = "foo";
 // const char* txt    = "hello";
 
-static void _publish_ack_cb(const char* guid, const char* error, void* closure) {
+//static void _publish_ack_cb(const char* guid, const char* error, void* closure) {
     // TODO: delete object from waiting list, so we can check if some object didn't published for a long time
     //std::cout << "#Ack#, " << guid << std::endl;
     // myPubMsgInfo* pubMsg = (myPubMsgInfo*)closure;
     // printf("Ack handler for message ID=%s Data=%.*s GUID=%s - ", pubMsg->ID, pubMsg->size, pubMsg->payload, guid);
-    if (error != NULL) {
-        std::cout << "Error: " << error << std::endl;
-        done = true;    // TODO: locking
-    }
+//    if (error != NULL) {
+//        std::cout << "Error: " << error << std::endl;
+//        done = true;    // TODO: locking
+//    }
     // free(pubMsg);    // This is a good place to free the pubMsg info since we no longer need it
     // Notify the main thread that we are done. This is not the proper way and you should use some locking.
     // done = true;
-}
+//}
```

Comment:
there other commented code near, so it's better to use `/*…*/` here of even `#ifdef BLA_BLA_SYNC` so we can revert it fast

Reply:
@zxcat `/*...*/` - agree, also in another places I should comment, not delete. but not now.
option SYNC/ASYNC - may be, but not now.